testslide/runner.py

```diff
@@ -319,12 +320,18 @@ def __init__(
         self.seed = seed
         self.focus = focus
         self.fail_fast = fail_fast
+        self.fail_if_focused = fail_if_focused
         self.names_text_filter = names_text_filter
         self.names_regex_filter = names_regex_filter
         self.names_regex_exclude = names_regex_exclude
         self.quiet = quiet
 
     def _run_example(self, example):
+        if example.focus and self.fail_if_focused:
+            raise AssertionError(
+                "Not running focused example: "
+                "{} as --fail-if-focused was specified".format(example.name)
```

Comment:
The test runner should already cover telling the test name, context and other stuff. This can be simplified for something like *Focused example not allowed with --fail-if-focused. Please remove the focus to allow the test to run.*